src/PostProcess/RenderPipeline/babylon.postProcessRenderEffect.ts

```diff
@@ -27,59 +26,28 @@ module BABYLON {
 
             this._postProcesses = {};
 
-            this._renderPasses = {};
             this._renderEffectAsPasses = {};
         }
 
         public get isSupported(): boolean {
             for (var index in this._postProcesses) {
-                if (!this._postProcesses[index].isSupported) {
-                    return false;
+                for(var ppIndex in this._postProcesses[index]){
+                    if (!this._postProcesses[index][ppIndex].isSupported) {
+                        return false;
+                    }
                 }
             }
             return true;
         }
 
         public _update(): void {

```

Comment:
This method is now empty. There are a couple places where this was called but it had no effect in the past as nothing was using postProcessRenderPass. Potentially logic will want to be added here in the future so I left it in.